tests/view/renderer.js

```diff
@@ -1216,6 +1217,40 @@ describe( 'Renderer', () => {
 			} ).to.throw( CKEditorError, /^view-renderer-filler-was-lost/ );
 		} );
 
+		// #1014.
+		it( 'should create inline filler in newly created dom nodes', () => {
```

Comment:
I wanted comment that perhaps this test should be split into two separate cases but its first part worked well before. As @scofalik commented in #1014 all these conditions need to be met.